common/infra-virt.function

```diff
@@ -48,7 +48,7 @@ upload_logs() {
     source ~/openrc
     BUILD_PLATFORM=${BUILD_PLATFORM:-"unknown_platform"}
     CONTAINER=${CONTAINER:-"unknown_platform"}
-    for path in /var/lib/edeploy/logs /var/log  /var/lib/jenkins/jobs/puppet/workspace; do
+    for path in /var/lib/edeploy/logs /var/log  /var/lib/jenkins/jobs/puppet/workspace /var/lib/jenkins/jobs/builds/1; do
```

Comment:
I was expecting something like /var/lib/jenkins/jobs/sanity/workspace

What will happend if the directory is missing?